Wox.Plugin.Runner/Wox.Plugin.Runner.csproj

```diff
@@ -1,15 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0-windows</TargetFramework>
     <OutputType>Library</OutputType>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <Nullable>enable</Nullable>

```

Comment:
I also enabled the [nullable references stuff](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references), which makes it a lot easier to spot potential null pointer exceptions at compile time.